python/graphscope/client/session.py

```diff
@@ -707,6 +726,10 @@ def __repr__(self):
     def __str__(self):
         return repr(self)
 
+    @property
+    def cluster_instance_id(self):
+        return self._launcher.instance_id
```

Comment:
It is not always available, if not running on K8s.

Why we need to expose the internal id in session API?

Reply:
Deleted.